views/report.dt

```diff
@@ -468,7 +468,9 @@ html(lang="en")
 
 				if(data.expressionHistory)
 				{
-					const emojisMap = data.expressionHistory.flatMap(e => e.emojis);
+					const emojisMap = data.expressionHistory
+						.flatMap(e => e.emojis)
+						.sort((a,b) => b.shares - a.shares);
```

Comment:
why not be more explicit?
```suggestion
						.sort((a, b) => b.shares > a.shares);
```

Reply:
That doesn't work in Javascript. The `sort` prototype in Javascript uses a comparator function instead of a boolean function. See here https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Array/sort